src/components/Layout/HomeContent.js

```diff
@@ -249,30 +250,31 @@ export function HomeContent() {
           </FullBleed>
           <Center>
             <Para>
-              React is also an architecture. Frameworks that implement it let
-              you fetch data in asynchronous components that run on the server
-              or even during the build. Read data from a file or a database, and
-              pass it down to your interactive components.
+              React também é uma arquitetura. Os frameworks que o implementam
+              permitem buscar dados em componentes assíncronos que rodam no
+              servidor ou até mesmo durante o build. Leia os dados de um arquivo
+              ou banco de dados e passe-os para seus componentes interativos.
             </Para>
             <div className="flex justify-start w-full lg:justify-center">
               <CTA
                 color="gray"
                 icon="framework"
                 href="/learn/start-a-new-react-project">
-                Get started with a framework
+                Comece com um framework
```

Comment:
Uma alternativa, mas não me importo tanto, seria:
```suggestion
                Inicie com um framework
```